introduction_to_amazon_algorithms/linear_learner_abalone/Linear_Learner_Regression_csv_format.ipynb

```diff
@@ -373,42 +357,50 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%time\n",
-    "linear.fit(inputs={'train': train_data, \n",
-    "            'validation': validation_data},job_name=job_name)"
+    "sagemaker.Session().delete_endpoint(linear_predictor.endpoint_name)\n",
+    "print(f\"deleted {linear_predictor.endpoint_name} successfully!\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Set up hosting for the model\n",
+    "# Appendix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Downloading the dataset\n",
     "\n",
-    "Once the training is done, we can deploy the trained model as an Amazon SageMaker real-time hosted endpoint. This will allow us to make predictions (or inference) from the model. Note that we don't have to host on the same insantance (or type of instance) that we used to train. Training is a prolonged and compute heavy job that require a different of compute and memory requirements that hosting typically do not. We can choose any type of instance we want to host the model. In our case we chose the ml.m4.xlarge instance to train, but we choose to host the model on the less expensive cpu instance, ml.c4.xlarge. The endpoint deployment can be accomplished as follows:\n"
+    "We are downloading the dataset in the original libvsm format, more info about this format can be found [here](https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression.html)."
```

Comment:
Should be s3?

Reply:
appendix isn't supposed to work in network isolation model